### ISSUE TITLE
Fix tour map sidebar scrolling

### ISSUE THREAD

**Issue by lederer** [azavea/cac-tripplanner#1162]
The fix requires a DOM change…

The tour map sidebar currently contains the following DOM:
```
.directions-results > .directions-list > .tours
```

Essentially we need to remove the intermediate `.directions-list` when in tour mode, so the corresponding DOM would look like:
```
.directions-results > .tours
```

Alternatively, tour mode could replace `.directions-results` entirely with a new tour-mode container. Something like this:
```
.tour-results > .tours
```

Either solution should work, and would just require some additional CSS tweaks from me once it's done.

**Comment by flibbertigibbet** [comment]
Pushed as a commit to #1126, which has been rebased to pick up the changes from #1161.

**Comment by flibbertigibbet** [comment]
Closed in #1168.
